samza-core/src/main/java/org/apache/samza/config/JobCoordinatorConfig.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.samza.config;
+
+import com.google.common.base.Strings;
+
+public class JobCoordinatorConfig extends MapConfig {
+  // TODO: Change this to job-coordinator.factory
+  private static final String JOB_COORDINATOR_FACTORY = "job.coordinator.factory";
+
+  public JobCoordinatorConfig(Config config) {
+    super(config);
+  }
+
+  public String getJobCoordinatorFactoryClassName() {
+    String jobCoordinatorFactoryClassName = get(JOB_COORDINATOR_FACTORY);
```

Comment:
I actually thought we are already using the config name at LinkedIn and it will become backward incompatible for those client. But I just checked. This is newly introduced config. So, I will change it now. Thanks for noticing it :)